app/src/main/res/values/strings.xml

```diff
@@ -161,6 +161,10 @@
     <string name="notification_pinned_site">Pinned to Firefox TV homescreen</string>
     <string name="notification_unpinned_site">Removed from Firefox TV homescreen</string>
 
+    <!-- TV Guide Channel onboarding screen -->
+    <string name="tv_onboarding_title">Now playing: the internet</string>
+    <string name="tv_onboarding_description">Watch videos from the most popular sites: news, sports, culture, music, and more. Scroll down. Select. Watch</string>
```

Comment:
Could you add a comment above each of these describing where it's used?  These get exported to our localizers for context.